recipes/astyle/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set name = "astyle" %}
+{% set version = "3.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/astyle/astyle/astyle%203.5/astyle-{{ version }}.tar.bz2
+  sha256: defc1cb7bfa863bec470e836c257439a1199ca2c8b13a652a327fd78c8f5923c
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+
+test:
+  commands:
+    - {{ name }} -h
```

The excess of jinja makes the recipe hard to read and doesn't really save us anything.

```suggestion
    - astyle -h
```